File: Bahasa C/PRAK502-2210817210013-Muhammad-ShandyDailami.c
#include <stdio.h>
#include <math.h>
int hitung(int nilai1, int nilai2){
    return nilai1 - nilai2;
}

int mutlak(int angka) {
    return fabs(angka);
}

int main() {
    int a, b, c, d, i = 0;
    while(i < 3) {
        scanf("%d",&a);
        scanf("%d",&c);
        scanf("%d",&b);
        scanf("%d",&d);
        int Hasil = hitung(a,b) + hitung(c,d); 
        printf("%d",mutlak(Hasil));
    i++;
    }
    return 0;
}

File: Python/PRAK502-2210817210013-Muhammad-ShandyDailami.py
def hitung(nilai1, nilai2):
    return int(nilai1) - int(nilai2)

def mutlak(angka):
    return abs(angka)

for i in range(3):
    x1, y1, x2, y2 = input().split()
    hasil = hitung(x1, x2) + hitung(y1, y2)
    print(f'{mutlak(hasil)}\n')